src/HotChocolate/MongoDb/src/Data/Extensions/MongoFilterVisitorContextExtensions.cs

```diff
@@ -10,20 +10,17 @@ public static class MongoFilterVisitorContextExtensions
     /// </summary>
     /// <param name="context">The context</param>
     /// <returns>The current scope</returns>
-    public static MongoDbFilterScope GetMongoFilterScope(
-        this MongoDbFilterVisitorContext context) =>
-        (MongoDbFilterScope)context.GetScope();
+    public static MongoDbFilterScope GetMongoFilterScope(this MongoDbFilterVisitorContext context)
+        =>
+            (MongoDbFilterScope)context.GetScope();
 
     /// <summary>
     /// Tries to build the query based on the items that are stored on the scope
     /// </summary>
     /// <param name="context">the context</param>
-    /// <param name="query">The query that was build</param>
     /// <returns>True in case the query has been build successfully, otherwise false</returns>
-    public static bool TryCreateQuery(
-        this MongoDbFilterVisitorContext context,
-        [NotNullWhen(true)] out MongoDbFilterDefinition? query)
+    public static MongoDbFilterDefinition CreateQuery(this MongoDbFilterVisitorContext context)
     {
-        return context.GetMongoFilterScope().TryCreateQuery(out query);
+        return context.GetMongoFilterScope().CreateQuery();
```

Comment:
Do we really need to break the API for 12 or can we go with the old one even if we no longer need the TryCreate? we could do the break in the 13 code base. We had now so many binary issues in user projects because we thought nobody will use this or that and every time we ran into an issue with this.